vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
```

Comment:
Should "Entando operator" be converted to a proper noun with both terms capitalized?

Reply:
this is an issue for the style guide. Here I kept it lowercase since it is not a particular instance. I think if the sentence still makes sense without the word Entando, it can be left lowercase. But we should discuss further for style guide. 

In this case, necessary is about the same as needed in meaning and I think the simpler and shorter is better.  Another issue for style guide.

Reply:
gotcha. i love that rule - "if the sentence still makes sense without the word Entando, it can be left lowercase." we should definitely formally adopt that!
i can see that; it struck my ear a little funny but i agree about simpler and shorter.

Reply:
I think if we're calling out the specific Entando Operator functionality (which we are in this case although it's not obvious) operator should be capitalized. It would look weird to say "...is needed for the operator to create..." because there can be many operators in a cluster or project and it's only our Entando Operator that would handle this task so we are pointing to our instance.